File: DownloadLib/src/main/java/com/kronos/download/DownloadSettingConfig.java
package com.kronos.download;

/**
 * Created by zhangyang on 2017/6/15.
 */

public class DownloadSettingConfig {
    private boolean autoDownload = false;
    private String fileSuffix = "";

    public boolean isAutoDownload() {
        return autoDownload;
    }

    public DownloadSettingConfig setAutoDownload(boolean autoDownload) {
        this.autoDownload = autoDownload;
        return this;
    }

    public String getFileSuffix() {
        return fileSuffix;
    }

    public DownloadSettingConfig setFileSuffix(String fileSuffix) {
        this.fileSuffix = fileSuffix;
        return this;
    }
}

File: DownloadLib/src/main/java/com/kronos/download/DownloadSettingConfig.kt
package com.kronos.download

/**
 * Created by zhangyang on 2017/6/15.
 */

class DownloadSettingConfig {
    private var autoDownload = false
    internal var fileSuffix = ""

    fun isAutoDownload(): Boolean {
        return autoDownload
    }

    fun setAutoDownload(autoDownload: Boolean): DownloadSettingConfig {
        this.autoDownload = autoDownload
        return this
    }

    fun getFileSuffix(): String {
        return fileSuffix
    }

    fun setFileSuffix(fileSuffix: String): DownloadSettingConfig {
        this.fileSuffix = fileSuffix
        return this
    }
}
